pkgs/development/python-modules/icoextract/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, pefile, pillow}:
+
+buildPythonPackage rec {
+  pname = "icoextract";
+  version = "0.1.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    sha256 = "sha256-orpFBnTByeS6I5gY1tkh5Xm8lvUjEm7IVrf82TA+9n4=";
+  };
+
+  propagatedBuildInputs = [
+    pefile
+    pillow
+  ];
+
+  # no tests in pypi tarball
+  doCheck = false;
```

Comment:
Then the package must be fetched from github https://github.com/jlu5/icoextract/tree/master/tests

Reply:
Will do.